ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/documentation/devices/SITE2-LSR2.md

```diff
@@ -147,7 +147,10 @@ vlan internal order ascending range 1006 1199
 
 | Interface | Description | Type | Channel Group | IPv6 Address | VRF | MTU | Shutdown | ND RA Disabled | Managed Config Flag | IPv6 ACL In | IPv6 ACL Out |
 | --------- | ----------- | ---- | --------------| ------------ | --- | --- | -------- | -------------- | -------------------| ----------- | ------------ |
-| Ethernet3 | P2P_LINK_TO_SITE1-LSR2_Ethernet3 | routed | - | - | default | 9178 | false | - | *- | - | - |
+| Ethernet3 | P2P_LINK_TO_SITE1-LSR2_Ethernet3 | routed | - | - | default | 9178 | false | - | - | - | - |
+| Ethernet12 | P2P_LINK_TO_SITE2-LER1_Port-Channel11 | *routed | 12 | *- | *default | *9178 | *false | *- | *- | *- | *- |
```

Comment:
are the `*` expected here?

Reply:
They are - it is because of the inheritance from the port-channel interface - check the * below the table. Strictly speaking these tables are not correct, since the Ethernet interface is not routed or with a certain MTU. It is the associated port-channel that has these settings. But this is the style we have for now.

Reply:
ah yes - my bad - I was just skimming the commits and so that the star in line 150 was removed. Thanks for the answer